### DISCUSSION TITLE
使用 popover 时，如何实现点击 popover 中的按钮后关闭 popover，并且保持实现点击 popover 外部时关闭popover

### DISCUSSION THREAD

**Discussion by heneyin** [element-plus/element-plus#13604] [Q&A] [answered]
使用 popover 时，如何实现点击 popover 中的按钮后关闭 popover，并且保持实现点击 popover 外部时关闭popover

**Comment by kaine0923** [comment, marked as answer]
try this [Element Plus Playground](https://element-plus.run/#eyJzcmMvQXBwLnZ1ZSI6IjxzY3JpcHQgc2V0dXAgbGFuZz1cInRzXCI+XG5pbXBvcnQgeyByZWYsdW5yZWYsIHZlcnNpb24gYXMgdnVlVmVyc2lvbiB9IGZyb20gJ3Z1ZSdcbmltcG9ydCB7IHZlcnNpb24gYXMgZXBWZXJzaW9uIH0gZnJvbSAnZWxlbWVudC1wbHVzJ1xuaW1wb3J0IHsgRWxlbWVudFBsdXMgfSBmcm9tICdAZWxlbWVudC1wbHVzL2ljb25zLXZ1ZSdcbmltcG9ydCB7IENsaWNrT3V0c2lkZSBhcyB2Q2xpY2tPdXRzaWRlIH0gZnJvbSAnZWxlbWVudC1wbHVzJ1xuY29uc3QgYnV0dG9uUmVmID0gcmVmKClcbmNvbnN0IHBvcG92ZXJSZWYgPSByZWYoKVxuY29uc3QgaXNWaXNpYmxlID0gcmVmKGZhbHNlKVxuZnVuY3Rpb24gY2xvc2UoKSB7XG4gIGlzVmlzaWJsZS52YWx1ZSAgPSBmYWxzZVxufVxuY29uc3Qgb25DbGlja091dHNpZGUgPSAoKSA9PiB7XG4gIGNsb3NlKClcbn1cblxuY29uc3QgbXNnID0gcmVmKCdIZWxsbyBXb3JsZCEnKVxuPC9zY3JpcHQ+XG5cbjx0ZW1wbGF0ZT5cbiAgPGgxPnt7IG1zZyB9fTwvaDE+XG4gIDxlbC1pbnB1dCB2LW1vZGVsPVwibXNnXCIgLz5cblxuICA8cD5cbiAgICA8ZWwtaWNvbiBjb2xvcj1cInZhcigtLWVsLWNvbG9yLXByaW1hcnkpXCI+PEVsZW1lbnRQbHVzIC8+PC9lbC1pY29uPlxuICAgIEVsZW1lbnQgUGx1cyB7eyBlcFZlcnNpb24gfX0gKyBWdWUge3sgdnVlVmVyc2lvbiB9fVxuICA8L3A+XG4gIDxlbC1idXR0b24gcmVmPVwiYnV0dG9uUmVmXCIgQGNsaWNrPVwiaXNWaXNpYmxlID0gdHJ1ZVwiIHYtY2xpY2stb3V0c2lkZT1cIm9uQ2xpY2tPdXRzaWRlXCJcbiAgICA+Q2xpY2sgbWU8L2VsLWJ1dHRvblxuICA+XG5cbiAgPGVsLXBvcG92ZXJcbiAgICByZWY9XCJwb3BvdmVyUmVmXCJcbiAgICA6dmlydHVhbC1yZWY9XCJidXR0b25SZWZcIlxuICAgIHRyaWdnZXI9XCJjbGlja1wiXG4gICAgdGl0bGU9XCJXaXRoIHRpdGxlXCJcbiAgICB2aXJ0dWFsLXRyaWdnZXJpbmdcbiAgICA6dmlzaWJsZT1cImlzVmlzaWJsZVwiXG4gID5cbiAgICA8ZGl2PlxuICAgICAgPHNwYW4+IFNvbWUgY29udGVudCA8L3NwYW4+XG4gICAgICA8ZWwtYnV0dG9uIEBjbGljaz1cImNsb3NlXCI+Y2xvc2U8L2VsLWJ1dHRvbj5cbiAgICA8L2Rpdj5cbiAgPC9lbC1wb3BvdmVyPlxuPC90ZW1wbGF0ZT5cbiIsImltcG9ydC1tYXAuanNvbiI6IntcbiAgXCJpbXBvcnRzXCI6IHt9XG59IiwidHNjb25maWcuanNvbiI6IntcbiAgXCJjb21waWxlck9wdGlvbnNcIjoge1xuICAgIFwiYWxsb3dKc1wiOiB0cnVlLFxuICAgIFwiY2hlY2tKc1wiOiB0cnVlLFxuICAgIFwianN4XCI6IFwicHJlc2VydmVcIixcbiAgICBcInRhcmdldFwiOiBcIkVTTmV4dFwiLFxuICAgIFwibW9kdWxlXCI6IFwiRVNOZXh0XCIsXG4gICAgXCJtb2R1bGVSZXNvbHV0aW9uXCI6IFwiQnVuZGxlclwiLFxuICAgIFwiYWxsb3dJbXBvcnRpbmdUc0V4dGVuc2lvbnNcIjogdHJ1ZSxcbiAgICBcInR5cGVzXCI6IFtcImVsZW1lbnQtcGx1cy9nbG9iYWwuZC50c1wiXVxuICB9LFxuICBcInZ1ZUNvbXBpbGVyT3B0aW9uc1wiOiB7XG4gICAgXCJ0YXJnZXRcIjogMy4zXG4gIH1cbn1cbiIsIl9vIjp7fX0=)
